articles/quickstart/native/ionic4/01-login.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Login
+default: true
+description: This tutorial demonstrates how to add user login to an Ionic 4 application using Auth0.
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - ionic4
+github:
+  path: 01-Login
+contentType: tutorial
+useCase: quickstart
+---
+
+<%= include('../_includes/_getting_started', { library: 'Ionic' }) %>
+
+<%= include('../../../_includes/_callback_url') %>
+
+The **Callback URL** to be used for your application includes your app's package ID which is found in the `config.xml` file for your app.
+
+Go to the <a href="${manage_url}/#/applications/${account.clientId}/settings">Application Settings</a> section in your Auth0 dashboard and set your **Callback URL** in the **Allowed Callback URLs** box.
+
+If you are following along with the sample project you downloaded from the top of this page, you should set the **Allowed Callback URL** to
```

Comment:
I don’t think we need to mention the sample app.  If we show them how to retrieve `YOUR_PACKAGE_ID`, they can do it in their own app or the sample app.